operations/iscsi_sbps/iscsi_sbps.md

```diff
@@ -0,0 +1,198 @@
+  | o- pscsi .................................................................................................. [Storage Objects: 0]
+  | o- ramdisk ................................................................................................ [Storage Objects: 0]
+  | o- rbd .................................................................................................... [Storage Objects: 0]
+  o- iscsi ............................................................................................................ [Targets: 1]
+  | o- iqn.2023-06.csm.iscsi:ncn-w002 .................................................................................... [TPGs: 1]
+  |   o- tpg1 .................................................................................................. [gen-acls, no-auth]
+  |     o- acls .......................................................................................................... [ACLs: 0]
+  |     o- luns .......................................................................................................... [LUNs: 4]
+  |     | o- lun0  [fileio/ad7d9e9736a1b8a (/var/lib/cps-local/boot-images/f1d6c8fe-32e2-420a-a051-377e34a8bd8a/rootfs) (default_tg_pt_gp)]
+  |     | o- lun1  [fileio/cd5c6354b0d12d2 (/var/lib/cps-local/boot-images/6e993608-068a-4f6d-8d3f-b904ff7d3602/rootfs) (default_tg_pt_gp)]
+  |     | o- lun2  [fileio/059c573240acc07 (/var/lib/cps-local/boot-images/1681d5d6-bfaf-41e6-9dd4-2cc355314476/rootfs) (default_tg_pt_gp)]
+  |     | o- lun3  [fileio/553a9957f5efcbf (/var/lib/cps-local/boot-images/c434edc1-8080-43c5-8393-4ab831d9eb00/rootfs) (default_tg_pt_gp)]
+  |     o- portals .................................................................................................... [Portals: 3]
+  |       o- 10.102.193.23:3260 ............................................................................................... [OK]
+  |       o- 10.252.1.8:3260 .................................................................................................. [OK]
+  |       o- 10.253.0.4:3260 .................................................................................................. [OK]
+  o- loopback ......................................................................................................... [Targets: 0]
+  o- vhost ............................................................................................................ [Targets: 0]
+  o- xen-pvscsi ....................................................................................................... [Targets: 0]
+```
+In above 'targetcli ls' command output, we can see that four fileio backing store is created for four rootfs images and four iSCSI LUNs created which has the 'rootfs' id being mapped and ready for rootfs image projection. 
+
+Sample initiator node snippet after image projection:
+```bash

```

Comment:
Again, leave off the `bash`, this isn't a bash script, it is command output and calling it a bash script is causing all the pipes(|) to be displayed in red.

Reply:
done